src/brick/tmpl/brick.php

```diff
@@ -0,0 +1,46 @@
+<script src="https://api.paymentwall.com/brick/brick.1.3.js"></script>
+<div id="payment-form-container"></div>
```

Comment:
The better way, If you want to use Brick default form, you should move Brick form content into an iframe.
So, you no need care the page style